### ISSUE TITLE
GraphQL Error: Schema must contain unique named types but contains multiple types named

### ISSUE THREAD

**Issue by Jones-S** [spicywebau/craft-neo#272]
#### Description
When querying a neo field like (pageBuilder = Neo Field), where I have a blocktype `standardSection` with a title:

```
query Page($slug: [String]) {
  entries(section: ["pages"], slug: $slug) {
    title
    ...on pages_pages_Entry {
    	pageBuilder {
        ...on pageBuilder_standardSection_BlockType {
          title
        }
      }
    }
  }
}
```

I would get this in return:

```
"debugMessage": "Schema must contain unique named types 
but contains multiple types named \"pageBuilder_standardSection_BlockType\". 
Make sure that `resolveType` function of abstract type \"pageBuilder_NeoField\" 
returns the same type instance as referenced anywhere else within the schema 
(see http://webonyx.github.io/graphql-php/type-system/#type-registry).",
```

I realize that this problem was addressed in another post already:
https://github.com/spicywebau/craft-neo/issues/268

Indeed downgrading to 3.3.2 worked, but I also read here:
https://github.com/craftcms/cms/issues/5067
That this was reported to Craft already.
They say that 3rd party plugins have to adapt their code:
https://github.com/craftcms/cms/issues/5067#issuecomment-539519135

Is this the case, or how can I fix this error in 3.3.7?
( I can use the fix with the schema, adding this to my query:
```
__schema {
    types {
      name
    }
  }
```
But that feels a bit hacky 😅

#### Steps to reproduce
1. Use Neo field
2. Query it in graphql

#### Other information
- Neo version: 2.5.3
- Craft version: 3.3.9

**Comment by pvldigital** [comment]
Hi @Jones-S,

The changes in https://github.com/craftcms/cms/issues/5067#issuecomment-539519135 has been made for neo. It was released in v2.5.4. Can you try upgrading and see if the error still occurs?

**Comment by Jones-S** [comment]
Oh damn. Deeply sorry for that. Haven't checked if there was already another update. Thanks it works. 👍
